### PR TITLE
Fixtures

### PR DESCRIPTION
**Pull request by kripton**
Hi, I had some time on an plane and worked on some fixtures. For details, please see commit messages. Feedback welcome

### CODE REVIEW

File: resources/fixtures/American_DJ/American-DJ-Vizi-Wash-LED-108.qxf [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  <Capability Min="100" Max="119">Internal program 1</Capability>
+  <Capability Min="120" Max="139">Internal program 2</Capability>
+  <Capability Min="140" Max="159">Internal program 3</Capability>
+  <Capability Min="160" Max="179">Internal program 4</Capability>
+  <Capability Min="180" Max="199">Internal program 5</Capability>
+  <Capability Min="200" Max="219">Internal program 6</Capability>
+  <Capability Min="220" Max="239">Internal program 7</Capability>
+  <Capability Min="240" Max="255">Music controlled program</Capability>
+ </Channel>
+ <Mode Name="MODUS1">
```
Comment (mcallegari) [inline resources/fixtures/American_DJ/American-DJ-Vizi-Wash-LED-108.qxf]: Use global physical

Comment (kripton) [reply resources/fixtures/American_DJ/American-DJ-Vizi-Wash-LED-108.qxf]: Thanks for the review. Changes happily done :D

**Comment by kripton** [comment]
Thanks for the review. I've updated the PR and pushed just as a safety measure before the flight. I'll finish the work one home :)

File: resources/fixtures/Litecraft/Litecraft-WashX.36.qxf [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  <Channel Number="1">Dimmer</Channel>
+  <Channel Number="2">Control</Channel>
+  <Channel Number="3">Red</Channel>
+  <Channel Number="4">Green</Channel>
+  <Channel Number="5">Blue</Channel>
+  <Channel Number="6">White</Channel>
+ </Mode>
+ <Physical>
+  <Bulb Type="" Lumens="0" ColourTemperature="0"/>
+  <Dimensions Weight="0" Width="0" Height="0" Depth="0"/>
```
Comment (mcallegari) [inline resources/fixtures/Litecraft/Litecraft-WashX.36.qxf]: Invalid. Please fix.

Comment (kripton) [reply resources/fixtures/Litecraft/Litecraft-WashX.36.qxf]: True, I didn't have the manual at that time (impossible to find online ...). Fixture is completed now

**Comment by kripton** [comment]
So, this should be everything now. Feel free to review and point out things to be fixed. If all looks good, please merge :)

File: resources/fixtures/Varytec/Varytec-BAT.PAR-6-RGBUV.qxf [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+ <Type>Color Changer</Type>
+ <Channel Name="Master Dimmer" Preset="IntensityMasterDimmer"/>
+ <Channel Name="Red" Preset="IntensityRed"/>
+ <Channel Name="Green" Preset="IntensityGreen"/>
+ <Channel Name="Blue" Preset="IntensityBlue"/>
+ <Channel Name="UV" Preset="IntensityUV"/>
+ <Channel Name="Strobe" Preset="ShutterStrobeSlowFast"/>
+ <Channel Name="Colour Macros">
+  <Group Byte="0">Colour</Group>
+  <Capability Min="0" Max="4">Keine Fuktion</Capability>
```
Comment (kripton) [inline resources/fixtures/Varytec/Varytec-BAT.PAR-6-RGBUV.qxf]: Sorry, seems to be in german here. Working on it ...

Comment (kripton) [reply resources/fixtures/Varytec/Varytec-BAT.PAR-6-RGBUV.qxf]: Done

**Comment by mcallegari** [comment]
Many thanks! :+1:

**Pull request merged**